src/components/job-row.tsx

```diff
@@ -106,9 +107,25 @@ export function buildJobRow(
   showDetailView: (jobId: string) => void,
   reload: () => void
 ): JSX.Element {
+  const inputFile = job.job_files.find(
+    jobFile => jobFile.file_format === 'input' && jobFile.file_path
+  );
+
   const cellContents: React.ReactNode[] = [
-    <a onClick={() => showDetailView(job.job_id)}>{job.name}</a>,
-    job.input_filename,
+    <Link onClick={() => showDetailView(job.job_id)}>{job.name}</Link>,
+    inputFile ? (
+      <Link
+        onClick={() => {
+          app.commands.execute('docmanager:open', {
+            path: inputFile.file_path
+          });
+        }}
+      >
+        {job.input_filename}
+      </Link>
+    ) : (
+      job.input_filename
```

Comment:
Is `job.input_filename` by itself a valid `React.ReactNode`? I tried something similar to this in #265, but got an error.

Reply:
From React declaration:

```
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    /**
     * @deprecated Use either `ReactNode[]` if you need an array or `Iterable<ReactNode>` if its passed to a host component.
     */
    interface ReactNodeArray extends ReadonlyArray<ReactNode> {}
    type ReactFragment = {} | Iterable<ReactNode>;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

`ReactNode` => `ReactChild` => `ReactText` => `string | number`.